### ISSUE TITLE
Jetpack temperature instant decreases when Me collide with AlienQueen

### ISSUE THREAD

**Issue by bayerchemistry** [colobot/colobot#997]

**Comment by tomangelo2** [comment]
There are more such situations, like landing on plants, wrecks

**Comment by DavivaD** [comment]
Decreases on any object with CrashSpheres.
Even Mines!

**Comment by RaptorParkowsky** [comment]
I can confirm that's common bug existing even in the original Colobot game. It's time to fix this.

**Comment by ixPost** [comment]
Yes, I think it's so that if you overheat and fall onto a building you can get off of it.
